clubs/README.md

```diff
@@ -385,3 +393,85 @@ after they finish.
 Demos easily get backlogged, so it's important to keep them moving fast. Each
 demo should be a minute long -- if they're running long, make an announcement
 that everyone will need to keep their demos under a minute.
+
+## Meeting Templates
+
+As previously mentioned, Hack Club meetings alternate between running workshops
+and free-form hacking (with demos). From here on out, we'll refer to meetings
+with workshops as _workshop days_ and meetings with free-form hacking and demos
+as _demo days_. We've created the below templates for running workshop and
```

Comment:
Totally fair and I agree with you – can you submit a pull request for this? One idea for another name is "hack days".